### PR TITLE
feat: show last used pictograms in a separate tab

### PR DESCRIPTION
**Pull request by yohanboniface**
This feature was planned in the initial rework of the pictogram form UI, but not yet done. Some recent discussion in the OSM French forum reactivated the need for it.

cf https://forum.openstreetmap.fr/t/marker-and-marker-colors/21051

image

### CODE REVIEW

**Comment by davidbgk** [comment]
Capture d’écran, le 2024-02-12 à 09 26 17

Capture d’écran, le 2024-02-12 à 09 26 04

Capture d’écran, le 2024-02-12 à 09 25 42

The wording is not consistent across my apps anyway 😅

**Comment by Aurelie-Jallut** [comment]
Yes, I'll opt for a dedicated tab. I'm curious to see what the preview of Emoji and pictos linked to a URL would look like.

image

Not sure for about the label, maybe Recent (Récents) or Used (utilisés) ?

image

**Comment by yohanboniface** [comment]
"Used" seems bit like "usagés" for me :p I'd go with "Recent" :)

File: umap/static/umap/js/umap.icon.js [hunk 1]
```diff
@@ -14,11 +17,22 @@ U.Icon = L.DivIcon.extend({
     }
   },
 
+  _setRecent: function (url) {
+    if (L.Util.hasVar(url)) return
+    if (url === this.map.options.default_iconUrl) return
```
Comment (davidbgk) [inline umap/static/umap/js/umap.icon.js:22]: To change with `U.DEFAULT_ICON_URL` [from here](https://github.com/umap-project/umap/pull/1609/files#diff-c616c7abb3defc7b125bc302e655bde24254137640b762135b0eefc61910ece7R142).

**Review by davidbgk — APPROVED** [review]

File: umap/static/umap/js/umap.icon.js (hunk 1 above)
Comment (yohanboniface) [reply umap/static/umap/js/umap.icon.js]: Done! :)

**Pull request merged**